src/MSALWrapper.Benchmark/BrokerBenchmark.cs

```diff
@@ -87,14 +73,7 @@ private IPCAWrapper BuildPCAWrapper(ILogger logger, Guid clientId, Guid tenantId
                     enablePiiLogging: false,
                     enableDefaultPlatformLogging: true)
 ;
-            if (useNativeBroker)
-            {
-                clientBuilder.WithBroker(new BrokerOptions(BrokerOptions.OperatingSystems.Windows));
-            }
-            else
-            {
-                clientBuilder.WithBroker();
-            }
+            clientBuilder.WithBroker(new BrokerOptions(BrokerOptions.OperatingSystems.Windows));

```

Comment:
Based on the PR description, it sounds like this call is obsolete/deprecated. Is that not true?

Reply:
oh mb i did not make it clear, starting from `4.61.0`, usage of `WithBroker` with no parameter is deprecated, it need to be used with the parameter `options`

Reply:
Yes, the managed broker (old broker api where we passed no parameters) is obsolete.

Reply:
Ah understood! And the only place we used parameterless method was in the benchmark class?

Reply:
yes thats the only place!